src/Parser/Core/Modules/Items/DeceiversGrandDesign.js

```diff
@@ -0,0 +1,126 @@
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+
+import Module from 'Parser/Core/Module';
+
+const debug = false;
+const GUIDING_HAND_DURATION = 120000;
+const PROC_EVENT_START_BUFFER = 5000;
+const PROC_EVENT_END_BUFFER = 1000;
+
+class DecieversGrandDesign extends Module {
+  healing = 0;
+  healingAbsorb = 0;
+  dgdTargetOne = null;
```

Comment:
It would be a lot cleaner if this was an array (but it's not important and I'm willing to ignore it).